docs/tutorials/remote_soc.rst

```diff
@@ -0,0 +1,209 @@
+
+For the full GDS-II results and intermediate build artifacts, you can install the EDA tools on your local system, and run the same Python build script with the :keypath:`option, remote` parameter set to ``False``. We are not returning the full results during this early beta period because we want to minimize bandwidth, and we believe that our public beta is currently best suited for rapid prototyping and design exploration.
+
+Adding an SRAM block
+--------------------
+
+A CPU core is not very useful without any memory. Indeed, a real system-on-chip would need quite a few supporting IP blocks to be useful in the real world. At the very least, you would want a SPI interface for communicating with external non-volatile memory, a UART to get data in and out of the core, a debugging interface, and a small on-die cache.
+
+In this tutorial, we'll take the first step by adding a small (2 kilobyte) SRAM block and wiring it to the CPU's memory interface. This will teach you how to import and place a hard IP block in your design.
+
+The open-source Skywater130 PDK does not currently include foundry-published memory macros. Instead, they have a set of OpenRAM configurations which are blessed by the maintainers. You can use `those configurations <https://github.com/VLSIDA/OpenRAM/tree/stable/technology/sky130>`_ to generate RAM macros from scratch if you are willing to install the `OpenRAM utility <https://github.com/VLSIDA/OpenRAM>`_, or you can `download pre-built files <https://github.com/VLSIDA/sky130_sram_macros>`_ which have been published under a permissive license. We will use the ``sky130_sram_2kbyte_1rw1r_32x512_8`` block in this example.
+
+Once you have a GDS and LEF file for your RAM macro, create a new directory called ``sram`` in same location as your PicoRV32 build files, and copy the macro files there. Then, create a Python script which describes the RAM macro in a format which can be imported by SiliconCompiler::
+
+    import siliconcompiler
+
+    def setup(chip):
+        # Core values.
+        design = 'sky130_sram_2k'
+        stackup = chip.get('option', 'stackup')
+
+        # Create library Chip object.
+        lib = siliconcompiler.Library(chip, design)
+        lib.set('output', stackup, 'gds', f'sram/sky130_sram_2kbyte_1rw1r_32x512_8.gds')
+        lib.set('output', stackup, 'lef', f'sram/sky130_sram_2kbyte_1rw1r_32x512_8.lef')
+        # Set the 'copy' field to True, to pull these files
+        # into the build directory during the 'import' task.
+        lib.set('output', stackup, 'gds', True, field='copy')
+        lib.set('output', stackup, 'lef', True, field='copy')
+
+    return lib
```

Comment:
I think there is an indent missing here

Reply:
You are correct. Sorry that I didn't actually follow this tutorial end-to-end before submitting it for review.